gapic/src/main/com/google/gapid/views/AboutDialog.java

```diff
@@ -81,12 +85,25 @@ public String getTitle() {
       protected Control createDialogArea(Composite parent) {
         Composite area = (Composite)super.createDialogArea(parent);
 
-        Composite container = createComposite(area, centered(new RowLayout(SWT.VERTICAL)));
-        container.setLayoutData(new GridData(SWT.CENTER, SWT.FILL, true, true));
+        GridLayout gridLayout = Widgets.withMargin(new GridLayout(2, false), 20, 5);
```

Comment:
nit: I've been static importing the static `Widgets` functions, which makes it a bit easier to read.